docs/protocols/protocol-mappings.md

```diff
@@ -0,0 +1,208 @@
+---
+sidebar_position: 5
+title: Protocol Mappings
+---
+
+# Protocol Mappings
+
+## Overview
+
+AdCP operations map to different protocol transports (MCP, A2A, HTTP) with consistent semantics. Each protocol adapter handles the mapping between AdCP's unified model and protocol-specific requirements.
+
+## Operation Types
+
+AdCP defines three operation types based on execution characteristics:
+
+### Synchronous Operations
+Return immediately (typically < 1 second):
+- `get_products`
+- `list_creative_formats`
+- `get_capabilities`
+- `get_targeting_capabilities`
+- `get_creatives`
+- `get_messages` (A2A)
+
+### Asynchronous Operations
+Long-running operations that return task/job IDs:
+- `create_media_buy`
+- `update_media_buy`
+- `add_creative_assets`
+- `update_package`
+- `approve_adaptation`
```

Comment:
this really should be a step in the add_creative_assets workflow, not a standalone tool/task